### PR TITLE
Feature/[SFA-145]: update settings modal and increase slippage tolerance

### PR DESCRIPTION
**Pull request by freitasgouvea**
[ ] Before opening a pull request, please read the [contributing guidelines](https://github.com/pancakeswap/pancake-frontend/blob/master/CONTRIBUTING.md) first
[ ] If your PR is work in progress, open it as `draft`
[ ] Before requesting a review, all the checks need to pass
[ ] Explain what your PR does

### CODE REVIEW

File: apps/web/src/components/Menu/GlobalSettings/SettingsModal.tsx [hunk 1]
```diff
@@ -158,7 +158,7 @@ const SettingsModal: React.FC<React.PropsWithChildren<InjectedModalProps>> = ({
               {chainId === ChainId.BSC && <GasSettings />}
             </Flex>
           </>
-        )}
+        )} */}
```
Comment (arthurmr96) [inline apps/web/src/components/Menu/GlobalSettings/SettingsModal.tsx:161]: You can remove this commented code, if we want in the future to get it back we can consult the git history

File: apps/web/src/components/Menu/GlobalSettings/SettingsModal.tsx [hunk 2]
```diff
@@ -265,7 +265,7 @@ const SettingsModal: React.FC<React.PropsWithChildren<InjectedModalProps>> = ({
                 onChange={(e) => setSmartRouterOpen(e.target.checked)}
                 scale="md"
               />
-            </Flex>
+            </Flex> */}
```
Comment (arthurmr96) [inline apps/web/src/components/Menu/GlobalSettings/SettingsModal.tsx:268]: You can remove this commented code, if we want in the future to get it back we can consult the git history

**Review by arthurmr96 — CHANGES_REQUESTED** [review]

File: apps/web/src/components/Menu/GlobalSettings/TransactionSettings.tsx [hunk 3]
```diff
@@ -97,35 +97,35 @@ const SlippageTabs = () => {
...
+            10%
           </Button>
           <Button
             mt="4px"
             mr="4px"
             scale="sm"
             onClick={() => {
               setSlippageInput('')
-              setUserSlippageTolerance(50)
+              setUserSlippageTolerance(2000)
```
Comment (luanpontolio) [inline apps/web/src/components/Menu/GlobalSettings/TransactionSettings.tsx:112]: Could it be a constant or an Env.?

**Review by luanpontolio — APPROVED** [review]

File: apps/web/src/components/Menu/GlobalSettings/TransactionSettings.tsx (hunk 3 above)
Comment (freitasgouvea) [reply apps/web/src/components/Menu/GlobalSettings/TransactionSettings.tsx:112]: Yes, of course, we could put it in the env or be a constant too, but I think that in this case is just the percentage that will appear on the button and it's only used here, so maybe it's better to keep it the same as the original project. Anyway, thanks @luanpontolio for the suggestion and we can improve these things in the next implementations

**Pull request merged**